src/v2/view-builder/internals/BaseView.js

```diff
@@ -46,6 +47,8 @@ export default View.extend({
         selector: '.siw-main-header',
         options: this.options,
       });
+    } else {
+      $(`#${Enums.WIDGET_CONTAINER_ID}`).addClass('no-beacon');
```

Comment:
I think this shall be implement in `BaseHeader`

Reply:
Not every view uses BaseHeader.

Reply:
@haishengwu-okta